libmscore/navigate.cpp

```diff
@@ -515,5 +515,187 @@ ChordRest* Score::prevMeasure(ChordRest* element, bool mmRest)
       return 0;
       }
 
+//---------------------------------------------------------
+//   nextElement
+//---------------------------------------------------------
+
+Element* Score::nextElement()
+      {
+      Element* e = getSelectedElement();
+      int staffId = e->staffIdx();
```

Comment:
I haven't been checking for this as religiously as I should as I review the code, but I notice that you aren't testing for null here.  Could be it will never happen that this function gets called with nothing selected, but doesn't hurt to check anyhow, as things might change later.